android/assets/jsons/translations/German.properties

```diff
@@ -6374,8 +6369,7 @@ The amount of ⍾Science you receive at the end is dependent on the ⍾Science g
 Defensive Pacts = Schutzpakte
```

Comment:
I picked whatever's short and sounds good. Since "defensive" has a translation though (Verteidigung), I'll update all my translations to use that, including of "defensive war".